RELEASE-NOTES.md

```diff
@@ -4,6 +4,61 @@ section: overview
+- **Slider:** Added limit for lowest possible value of slider ([#3660](https://github.com/patternfly/patternfly-react/pull/3660))
+- **Wizard:** Added typeto wizard context ([#3572](https://github.com/patternfly/patternfly-react/pull/3572))
+
+### Catalog view extension
+- **Catalog tile:** 
+  - Adjusted max width of header image ([#3628](https://github.com/patternfly/patternfly-react/pull/3628))
+  - Reduced padding between badge and logo ([#3644](https://github.com/patternfly/patternfly-react/pull/3644))
+  - Fixed styles that stopped working ([#3635](https://github.com/patternfly/patternfly-react/pull/3635))
+  - Fixed linting errors ([#3714](https://github.com/patternfly/patternfly-react/pull/3714))
+
+### Charts
+- **React charts:** Fixed linting errors ([#3713](https://github.com/patternfly/patternfly-react/pull/3713))
+
+### Table:
+- Added inline edit to table ([#3058](https://github.com/patternfly/patternfly-react/issues/3058))
+- Reverted IHeaderRow interface change ([#3746](https://github.com/patternfly/patternfly-react/pull/3746))
+- Fixed that column functions are not always equal ([#3612](https://github.com/patternfly/patternfly-react/pull/3612))
+- Added support for truncated column headers ([#3729](https://github.com/patternfly/patternfly-react/pull/3729))
+
+### Other:
+- **Build:**
+  - Updated circleci config for pricing ([#3654](https://github.com/patternfly/patternfly-react/pull/3654))
+  - Made change to use machine instance ([#3712](https://github.com/patternfly/patternfly-react/pull/3712))
+  - Fixed typo, added md lint to circleci ([#3692](https://github.com/patternfly/patternfly-react/pull/3692))
+- **Chore:** 
+  - Phrased non production components as beta ([#3663](https://github.com/patternfly/patternfly-react/pull/3663))
+  - Linted react topology ([#3721](https://github.com/patternfly/patternfly-react/pull/3721))
+- **Cypress:** Parallelized tests and run in PRs ([#3627](https://github.com/patternfly/patternfly-react/pull/3627))
+- **Docs:** 
+  - Updated table property descriptions ([#3745](https://github.com/patternfly/patternfly-react/pull/3745))
+  - Updated node version in readme ([#3741](https://github.com/patternfly/patternfly-react/pull/3741))
+  - Bumped gatsby ([#3657](https://github.com/patternfly/patternfly-react/pull/3657))
+  - Merged branch gatsby-theme-patternfly-org ([#3651](https://github.com/patternfly/patternfly-react/pull/3651))
+  - Merged patternfly-org branch ([#3619](https://github.com/patternfly/patternfly-react/pull/3619))
+- **Eslint:** Added react-hooks to eslint-plugin-patternfly-react ([#3636](https://github.com/patternfly/patternfly-react/pull/3636))
+- **Github:** Fixed broken link in issue template ([#3634](https://github.com/patternfly/patternfly-react/pull/3634))
+- **Icons:** Made change to use absolute import paths ([#3517](https://github.com/patternfly/patternfly-react/pull/3517))
```

Comment:
I would probably put this under "Chore"

Reply:
Which one?

Reply:
Icons?

Reply:
yes

Reply:
Github can probably go under "chore" as well.